src/performance_profiles.jl

```diff
@@ -81,7 +81,11 @@ function performance_profile(T :: Array{Float64,2}, labels :: Vector{AbstractStr
     if length(linestyles) > 0
       kwargs[:linestyle] = linestyles[s]
     end
-    Plots.plot!(rs[xidx], xs[xidx], t=:steppost, label=labels[s]; kwargs...)
+    if rs[1] > 0
+      Plots.plot!([0.0; rs[xidx]], [0.0; xs[xidx]], t=:steppost, label=labels[s]; kwargs...)
```

Comment:
What is the logic here?

Reply:
Sometimes there will be no 0 ratio for a solver, so the solver starts in the middle of nowhere, i.e., the glitch. This adds a (0,0) point when needed. I didn't want to modify the code much, so this was the simplest fix.